### PR TITLE
Add core weekly 40

### PR DESCRIPTION
**Pull request by matks**

### CODE REVIEW

File: news/_posts/core-weekly/2020-10-05-coreweekly-40-2020.md [hunk 1]
```diff
@@ -0,0 +1,227 @@
...
+
+## General messages
+
+Dear developers,
+
+The past week has been filled with lot of interesting events!
+
+First, [php8 Release Candidate 1](https://www.php.net/archive/2020.php#2020-10-01-4) has been delivered and is available for testing. Obviously we suggest helping the test of this version as this is the future of php.
+
+Secondly, [Hacktoberfest](https://hacktoberfest.digitalocean.com/) has started on the 1st of October. But the very first days of the events have been complicated: the event [experimented a huge spam contributions wave](https://hacktoberfest.digitalocean.com/hacktoberfest-update). The consequences of this spam are that the rules of Hacktoberfest changed on the 3rd of October: projects now need to opt-in for the event and Pull Requests must be either merged, approved or labeled 'hacktoberfest-accepted' to be accepted as a participation.
```
Comment (matks) [inline news/_posts/core-weekly/2020-10-05-coreweekly-40-2020.md]:
```suggestion
Secondly, [Hacktoberfest](https://hacktoberfest.digitalocean.com/) has started on the 1st of October. But the very first days of the event have been complicated: the event [experimented a huge spam contributions wave](https://hacktoberfest.digitalocean.com/hacktoberfest-update). The consequences of this spam are that the rules of Hacktoberfest changed on the 3rd of October: projects now need to opt-in for the event and Pull Requests must be either merged, approved or labeled 'hacktoberfest-accepted' to be accepted as a participation.
```

File: news/_posts/core-weekly/2020-10-05-coreweekly-40-2020.md [hunk 2]
```diff
@@ -0,0 +1,227 @@
...
+
+![Core Weekly banner](/assets/images/2018/12/banner-core-weekly.jpg)
+
+## General messages
+
+Dear developers,
+
+The past week has been filled with lot of interesting events!
+
+First, [php8 Release Candidate 1](https://www.php.net/archive/2020.php#2020-10-01-4) has been delivered and is available for testing. Obviously we suggest helping the test of this version as this is the future of php.
```
Comment (PierreRambaud) [inline news/_posts/core-weekly/2020-10-05-coreweekly-40-2020.md]:
```suggestion
First, [php8 Release Candidate 1](https://www.php.net/archive/2020.php#2020-10-01-4) has been delivered and is available for testing. Obviously, we suggest helping the test of this version as this is the future of PHP.
```

File: news/_posts/core-weekly/2020-10-05-coreweekly-40-2020.md [hunk 3]
```diff
@@ -0,0 +1,227 @@
...
+
+Secondly, [Hacktoberfest](https://hacktoberfest.digitalocean.com/) has started on the 1st of October. But the very first days of the event have been complicated: the event [experimented a huge spam contributions wave](https://hacktoberfest.digitalocean.com/hacktoberfest-update). The consequences of this spam are that the rules of Hacktoberfest changed on the 3rd of October: projects now need to opt-in for the event and Pull Requests must be either merged, approved or labeled 'hacktoberfest-accepted' to be accepted as a participation.
```
Comment (PierreRambaud) [inline news/_posts/core-weekly/2020-10-05-coreweekly-40-2020.md]:
```suggestion
Secondly, [Hacktoberfest](https://hacktoberfest.digitalocean.com/) has started on the 1st of October. But the very first days of the event have been complicated: the event [experimented a huge spam contributions wave](https://hacktoberfest.digitalocean.com/hacktoberfest-update). The consequences of this spam are that the rules of Hacktoberfest changed on the 3rd of October: projects now need to opt-in for the event and Pull Requests must be either merged, approved, or labeled 'hacktoberfest-accepted' to be accepted as a participation.
```

File: news/_posts/core-weekly/2020-10-05-coreweekly-40-2020.md (hunk 2 above)
Comment (Progi1984) [reply news/_posts/core-weekly/2020-10-05-coreweekly-40-2020.md]:
```suggestion
First, [PHP 8.0 Release Candidate 1](https://www.php.net/archive/2020.php#2020-10-01-4) has been delivered and is available for testing. Obviously, we suggest helping the test of this version as this is the future of PHP.
```

**Review by Progi1984 — COMMENTED** [review]
Some feedback

**Review by PierreRambaud — APPROVED** [review]

File: news/_posts/core-weekly/2020-10-05-coreweekly-40-2020.md [hunk 4]
```diff
@@ -0,0 +1,227 @@
...
+
+![Core Weekly banner](/assets/images/2018/12/banner-core-weekly.jpg)
+
+## General messages
+
+Dear developers,
+
+The past week has been filled with lot of interesting events!
+
+First, [php8 Release Candidate 1](https://www.php.net/archive/2020.php#2020-10-01-4) has been delivered and is available for testing. Obviously, we suggest helping the test of this version as this is the future of PHP.
```
Comment (Progi1984) [inline news/_posts/core-weekly/2020-10-05-coreweekly-40-2020.md]:
```suggestion
First, [PHP 8.0 Release Candidate 1](https://www.php.net/archive/2020.php#2020-10-01-4) has been delivered and is available for testing. Obviously, we suggest helping the test of this version as this is the future of PHP.
```

**Review by PierreRambaud — APPROVED** [review]

**Review by Progi1984 — APPROVED** [review]

**Comment by Progi1984** [comment]
Thanks @matks

**Pull request merged**